script/Ciel/Day 3/03_01_CIEL03_3_3C.txt

```diff
@@ -352,48 +394,56 @@ That's why you are definitely a person who can atone."
 [sha:f31fbc8adca69f611556d71e6f52dc13b0c0567a]{
 -- Page 270, Offset 5539.
 -- ……そうなのだろうか。
-... I wonder if that's true.
+I wonder if that's really true...
 }
 [sha:d94113e325e15e8f7aee0ac68af24ef14474f8f3]{
 -- Page 270, Offset 5540.
 -- 俺は、俺を信じる事ができないけれど。
-But I can't trust myself.
+I may not be able to believe in myself.
 }
 [sha:141f651e475d62933af90b7139e599c7543c6c5f]{
 -- Page 270, Offset 5541.
 -- ……この人がそう言ってくれるなら、そうであるように生きたいと思った。
-... If this person says so, I want to live my life as if it were so.
+// Listen up, Shiki. Don't believe in yourself. Believe in me! Believe in the Ciel who believes in you!
+But if this person says I can... then I want to live my life as though it's possible.
 }
 [sha:1da4eeed8402f5e918e89e499a4840cf2320f796]{
 -- Page 271, Offset 5542.
 -- 「……わかんないな。俺、ひどい人間だよ。先輩が思ってるほどいいヤツじゃない」
-"...... I dunno. I'm a terrible person. I'm not as nice as you think I am."
+"I don't know about that...
+ I'm a terrible person, you know.
+ I'm not the nice guy you think I am."
 }
 [sha:f90f71085afd3b714dcfd0daf981b7a0b702398c]{
 -- Page 271, Offset 5543.
 -- 「そんなの簡単に分かりますよ。さっきですね、わたしちょっと感動しました。遠野くん、胸にすごい傷があるでしょ？」
-"That's easy to understand. Just now, I was a little moved. Tohno-kun, don't you have a huge scar on your chest?"
+"Oh, I have no trouble believing that much.
+ I was actually a little moved by what I saw earlier.
+ Tohno-kun, you have a large scar on your chest, right?"
 }
 [sha:07553858633605bbfb47c324ed248f9ab229cf2e]{
 -- Page 271, Offset 5544.
 -- 「あるけど……それがどうして？」
-"Yes, but... what about it?"
+"I do, but... what about it?"
 }
 [sha:b7ffcdc78175e4f22499a480c1dde0344e982d94]{
 -- Page 272, Offset 5545.
 -- 「あれだけの傷跡ということは、とても大きな事故だったんでしょう。爪跡というものは心を歪ませます。
-"With such a scar, it must have been a very serious accident. A scar can distort the mind.
+// 爪跡 explicitly different here, _maybe_ has more 'ravages', 'after-effects' connotations that pure 傷跡?
+"It must have been a very serious accident to leave you with a scar like that.
+ Lingering effects of things like that distort the mind.
 }
 [sha:9a8755fb707baf7790f450418af95d5490ae965d]{
 -- Page 272, Offset 5546.
 -- あんなに大きな、しかもまだ消えない傷だなんて、ちょっと普通じゃないですよね」
-It's a little unusual to have a scar that big and that still hasn't gone away."
+To receive a large wound like that, and for it to still be with you, is a pretty rare occurrence, isn't it?"
```

Comment:
be with you is pretty rare, isn't it?

Reply:
Not sure about dropping the comma after `with you,` it breaks the association slightly for me by orphaning the first part of the sentence

Reply:
Does feel like a lot of commas though - em dash it?
> To receive a large wound like that――and for it to still be with you――is pretty rare, isn't it?"

Reply:
makes more sense with em-dashes for sure